R/setup.R

```diff
@@ -284,6 +291,8 @@ metadata_add_locations <- function(dataset_id, location_data) {
   )
 
   write_metadata_dataset(metadata, dataset_id)
+
+  return(invisible(metadata))
```

Comment:
even though we saved to file, this will return the object silently, which enables us to test it